src/api/dataSource/NodeDataSource.ts

```diff
@@ -298,24 +310,25 @@ export class NodeDataSource implements NodeApi {
     selectedVersion: string,
     ipfsPath: string,
     hash: string,
-  ): ApiResponse<boolean> {
+  ): ApiResponse<InstallApplicationResponse> {
     try {
       const headers: Header | null = await createAuthHeader(
         JSON.stringify({
           selectedPackage,
           selectedVersion,
-        }),
-      );
-      const response: ResponseData<boolean> = await this.client.post<boolean>(
-        `${getAppEndpointKey()}/admin-api/install-application`,
-        {
-          application: selectedPackage,
-          version: selectedVersion,
-          url: ipfsPath,
           hash,
-        },
-        headers ?? {},
+        }),
       );
+      const response: ResponseData<InstallApplicationResponse> =
+        await this.client.post<InstallApplicationResponse>(
+          `${getAppEndpointKey()}/admin-api/install-application`,
+          {
+            contract_app_id: selectedPackage,
```

Comment:
`selectedPackage` does not indicate a connection with `contract_app_id`. 

Can you expand the update to the naming?

Reply:
selectedPackageId updated